org.osgi.test.assertj.cm/src/main/java/org/osgi/test/assertj/cm/configuration/ConfigurationAssert.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+
+package org.osgi.test.assertj.cm.configuration;
+
+import org.osgi.service.cm.Configuration;
+
+/**
+ * {@link Configuration} specific assertions Although this class is not final to
+ * allow Soft assertions proxy, if you wish to extend it, extend
+ * {@link AbstractConfigurationAssert} instead.
+ */
+public class ConfigurationAssert extends AbstractConfigurationAssert<ConfigurationAssert, Configuration> {
+
+	/**
+	 * Creates a new <code>{@link ConfigurationAssert}</code> to make assertions
+	 * on actual Configuration.
+	 * 
+	 * @param actual the Configuration we want to make assertions on.
+	 */
+	public ConfigurationAssert(Configuration actual) {
+		super(actual, ConfigurationAssert.class);
+	}
+
+	/**
+	 * An entry point for ConfigurationAssert to follow AssertJ standard
+	 * <code>assertThat()</code> statements.<br>
+	 * With a static import, one can write directly:
+	 * <code>assertThat(myConfiguration)</code> and get specific assertion with
+	 * code completion.
+	 * 
+	 * @param actual the Configuration we want to make assertions on.
+	 * @return a new <code>{@link ConfigurationAssert}</code>
+	 */
+	@org.assertj.core.util.CheckReturnValue
```

Comment:
Why do you have these annotation in here. We don't do anything with them. I assumed these were for assertj's testing of its API.

Reply:
just a leftover of the assertj-generator I used to get the stump. will do more cleanup.